### DISCUSSION TITLE
Using setQueryTimeout with JDBC and Sequel

### DISCUSSION THREAD

**Discussion by jpsikorra** [jeremyevans/sequel#2211] [General]
Hi, 

first of all thank you for creating and maintaining the sequel gem.

I was wondering if there is any way to use `setQueryTimeout` of JDBC for a statement in Sequel?

**Comment by jeremyevans** [comment]
There isn't a simple way currently.  Support could probably be added in a manner similar to how `setFetchSize` is currently implemented.
